package.nix

```
@@ -60,12 +63,26 @@ stdenv.mkDerivation {
     "wayland"
   ];
   buildPhase = ":";
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "Binary Ninja";
+      exec = "binaryninja";
+      icon = "logo";
```

I don't think the icon for binaryninja should be called `logo` in the global namespace of installed XDG icons. Let's rename the file.
```suggestion
      icon = "binaryninja";
```

---

That makes a lot of sense